### ISSUE TITLE
Lock Forestry

### ISSUE THREAD

**Issue by chameleon-tbn** [Quistar-LAB/TreeAnarchy#10]
I think your Option should have the either same setting as the 'Lock Forestry Resource' mod or should inform that there is another Mod active doing the same and pathing that one out...

ATM both conflict each other... 'Lock Forestry Resource' mod is on locked status and TreeAnarchy not... but 'Lock Forestry Resource' mod is used for locking

btw... it would be nice to have a shortcut for this to activate/deactivate and an visible indicator ingame (maybe recolor the Landscaping UI icon or Forest Brush UI button if that one is installed

**Comment by Quistar-LAB** [comment]
The old Lock Forestry mod used a Harmony prefix to reroute NaturalResourceManager::TreesModified function. I intend to do some fine tuning inside this method, thus I suggest to stop using Lock Forestry and uninstall it, Just use my version of Lock Forestry, where I used Transpilers to enable the same functionality. This ensures future compatibility.
As for your suggestion with a shortcut (an icon, or a keymapped shortcut), I have been considering it. Because I rarely use this function, and I only use this forestry when I'm creating the forestry industry, I felt that it wasn't necessary, and avoid UI clutter.
I think it best to get some more input before I take time to implement this function.

I'm currently working on improving the FPS of tree rendering. So far I've shaved off around 10ms of computation time per frame.
